docs/samples/ws-client.md

```diff
@@ -1,6 +1,37 @@
-<!-- Marker for build.py that there's no sample bundle. Remove this if you created one -->
-
 ## Using the WebSocket-client sample bundle
 
-No sample bundle for service `ws-client`.  
-[You can help us and create one!](https://github.com/codeoverflow-org/nodecg-io/blob/master/docs/docs/contribute.md)
+The WebSocket-client sample bundle in `samples/websocket-client` shows how to setup a simple WebSocket client that will ping a server every second.
+
+### Prerequisites
+
+-   Working NodeCG & nodecg-io installation
+
+For simplicity sake this sample will rely upon the werbsocket-server sample.
+
+### Configure the WebSocket-client sample bundle
+
+Please setup the WebSocket-server bundle first and then follow these steps:
+
+1. Start nodecg with nodecg-io installed. The websocket-server bundle is currently part of it so it should also be loaded.
+
+2. Go to the `nodecg-io` tab in the nodecg dashboard.
+
+3. Login using your password. If this is your first run, then enter the password with which you want to encrypt your configurations and credentials.
+
+4. Create a new ws-client service instance using the left upper menu.
+
+5. Enter the address of the sample server. This has to be a url following the pattern `http://localhost:<port>`.
```

Comment:
I didn't even know that we could use `http` and it would autocorrect it to `ws` but for avoiding confusion about the protocol schema we should use `ws`.
```suggestion
5. Enter the address of the sample server. This has to be a url following the pattern `ws://localhost:<port>`.
```